proto/dymension/rollapp/query.proto

```diff
@@ -93,78 +63,29 @@ message QueryAllRollappRequest {
 }
 
 message QueryAllRollappResponse {
-	repeated Rollapp rollapp = 1 [(gogoproto.nullable) = false];
+	repeated RollappSummary rollapp = 1 [(gogoproto.nullable) = false];
 	cosmos.base.query.v1beta1.PageResponse pagination = 2;
 }
 
 message QueryGetStateInfoRequest {
 	  string rollappId = 1;
 	  uint64 index = 2;
+		uint64 height = 3;
+		bool finalized = 4;
 }
 
 message QueryGetStateInfoResponse {
 	StateInfo stateInfo = 1 [(gogoproto.nullable) = false];
 }
 
 message QueryAllStateInfoRequest {
```

Comment:
what it does? bring all the commits of the rollapp??
thousands of commit?
why it's needed?